.github/workflows/build_and_deploy.yaml

```diff
@@ -15,23 +15,41 @@ jobs:
         id: tag-checker
         run: |
           (echo -n TRIGGER_ALLOWED= && echo 'print("${{ github.ref_name }}".split("_")[0] in "${{ vars.CURRENT_RELEASE }}")' | python3) >> "$GITHUB_OUTPUT"
-  docker-build:
+  docker-build-api-service:
     needs: check-tag
     if: needs.check-tag.outputs.ALLOWED_TAG ==  'True'
     runs-on: ubuntu-latest
     steps:
       - name: Login to docker hub
-        uses: docker/login-action@v2
+        uses: docker/login-action@v1
         with:
           username: ${{ vars.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-
       - name: Build docker image and push
-        uses: docker/build-push-action@v4
+        uses: docker/build-push-action@v2
         with:
+          file: ./api-service/Dockerfile  
           push: true
           tags: ${{ vars.DOCKERHUB_USERNAME }}/obsrv-api-service:${{ github.ref_name }}
 
+  docker-build-command-service:
+    needs: check-tag
+    if: needs.check-tag.outputs.ALLOWED_TAG ==  'True'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to docker hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build docker image and push
+        uses: docker/build-push-action@v2
+        with:
+          file: ./command-service/Dockerfile  
+          push: true
+          tags: ${{ vars.DOCKERHUB_USERNAME }}/obsrv-command-service:${{ github.ref_name }}
```

Comment:
rename the image repo to `flink-command-service` as it is being used currently for open source